de.tu_bs.cs.isf.e4cf.parts.project_explorer/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
+encoding//src/de/tu_bs/cs/isf/e4cf/parts/project_explorer/wizards/NewFilePage.java=UTF-8

```

Comment:
why does  the encoding have to specified for this one file explicitly? Just wondering.

Reply:
i did not understand that too. it has to do with the Regex, because when i first implemented it it gave me an error that we need utf8 encoding

Reply:
okay then